### ISSUE TITLE
Gate.io fetch_balance only loading spot balance.

### ISSUE THREAD

**Issue by jornx** [ccxt/ccxt#9773]
If I use fetch_balance() I only seem to get the balance of my spot accounts.
I have tried to set the default type to future with:

exchange.options['defaultType'] = 'future'

Same outcome, seems this isn't supported.

**Comment by frosty00** [comment]
Yes, only the spot api for gatev4 is supported currently, may plan to add the other market types asap. for now you can call them using the implicit api https://ccxt.readthedocs.io/en/latest/manual.html#implicit-api
